onnxruntime/python/tools/transformers/models/stable_diffusion/demo_utils.py

```diff
@@ -239,6 +239,7 @@ def parse_arguments(is_xl: bool, parser):
     )
     parser.add_argument("--nvtx-profile", action="store_true", help="Enable NVTX markers for performance profiling.")
     parser.add_argument("--seed", type=int, default=None, help="Seed for random generator to get consistent results.")
+    parser.add_argument("--deter-infer", action="store_true", help="deterministic inference.")
```

Comment:
nit: rename the option as --deterministic, and help info "use deterministic algorithms"

Reply:
thx, updated!